stdlib/public/core/FloatingPointParsing.swift.gyb

```diff
@@ -42,7 +42,7 @@ internal func _isspace_clocale(_ u: UTF16.CodeUnit) -> Bool {
 % end
 
 //===--- Parsing ----------------------------------------------------------===//
-extension ${Self} {
+extension ${Self} : LosslessStringConvertible {
```

Comment:
Do we have tests for every new conformance to LosslessStringConvertible?  We should.